src/coreclr/src/jit/morph.cpp

```diff
@@ -2668,6 +2504,9 @@ void Compiler::fgInitArgInfo(GenTreeCall* call)
         noway_assert(arg != nullptr);
         call->gtCallCookie = nullptr;
 
+        // TODO: Refactory proposal
+        // I am not good at naming.  The function name might be fgInitArgInfoHelpN below.
+
 #if defined(TARGET_X86)
```

Comment:
The patch is only the skeleton.

It needs to choose the precise name for the functions, but not the `fgInitArgInfoHelp` or `fgInitArgInfoHelpN` to replace the `#ifdef _TARGET_YOUR_ARCH` in the morph.cpp.

Please give me some advice about the naming.

Thanks,
Leslie Zhai